package.json

```diff
@@ -9,12 +9,12 @@
     "build:doc": "npm run clean && tsc -m es2015 -t es6 --moduleResolution node && esdoc -c esdoc.json",
     "clean": "rm -rf coverage doc lib",
     "prepare": "npm run clean && tsc -d",
-    "fmt": "prettier --single-quote --trailing-comma all --print-width 100 --write \"{src,test,examples}/**/*.ts\" && npm run test:lint -- --fix",
+    "fmt": "prettier --write \"{src,test,examples}/**/*.ts\" && npm run test:lint -- --fix",
     "test": "npm-run-all --parallel test:lint test:unit test:integrate",
     "test:browser": "karma start test/karma.conf.js",
     "test:cover": "npm run build:dist && istanbul cover _mocha -- lib/test/unit/*.test.js && open-cli coverage/lcov-report/index.html",
     "test:integrate": "mocha --require ts-node/register --timeout 20000 test/integrate/*.test.ts",
-    "test:lint": "eslint '{src,test}/**/*.ts'",
+    "test:lint": "eslint  \"{src,test,examples}/**/*.ts\" --quiet --fix",
```

Comment:
`test:lint` must not make automatic fixes (`--fix`) otherwise they will be done on the CI / Travis but not committed back into the repo.